lib/travis/build/script/node_js.rb

```diff
@@ -227,7 +227,8 @@ def npm_install(args)
               sh.if "$(vers2int `npm -v`) -ge $(vers2int #{NPM_CI_CMD_VERSION})" do
                 sh.if "-f npm-shrinkwrap.json -o -f package-lock.json" do
```

Comment:
Okayyy seems like it should be `-f npm-shrinkwrap.json || -f package-lock.json` instead..